modules/distribution/carbon-home/conf/worker/deployment.yaml

```diff
@@ -218,4 +218,8 @@ wso2.datasources:
         idleTimeout: 60000
         connectionTestQuery: SELECT 1
         validationTimeout: 30000
-        isAutoCommit: false
\ No newline at end of file
+        isAutoCommit: false
+
```

Comment:
Shall we remove these new lines

Reply:
This change has been overridden. _(See https://github.com/wso2/product-sp/blame/master/modules/distribution/carbon-home/conf/worker/deployment.yaml)_.